src/Articalizer.jsx

```diff
@@ -40,14 +33,19 @@ export default function Articalizer(props) {
                     flexDirection: "column",
                   }}
                 >
-                  <CardMedia
-                    component="div"
-                    sx={{
-                      // 16:9
-                      pt: "56.25%",
-                    }}
-                    image={article.article_img_url}
-                  />
+                  <CardActionArea component="div">
+                    <Link href={`http://localhost:5173/${article.article_id}`}>
```

Comment:
this needs to go to /article.id 
local host is the host your side!!!

Reply:
*slaps forehead* duh! Thank you Pippa! I spent so long trying to get this to work the way I wanted to I completely missed that it will only work for me haha

Can I ask, what is the link that ISNT localhost? That may sound like a really dumb question but I hadn't considered how you guys have actually been looking at our websites...?

Reply:
Oooooh do you mean just to replace the link with /article.article_id?? Because that now seems to be working without using localhost